src/core/ext/xds/xds_client.cc

```diff
@@ -2200,6 +2161,11 @@ void XdsClient::ResetBackoff() {
 }
 
 void XdsClient::NotifyOnErrorLocked(absl::Status status) {
+  const auto* node = bootstrap_->node();
+  if (node != nullptr) {
+    status = absl::Status(status.code(), absl::StrCat(
```

Comment:
would this capture everything we need from `status`?

Reply:
For example, if a payload was set on the status, this would miss that. cc @veblush

Reply:
We never add any payload for any of the statuses that we generate internally.